src/main/content/_assets/js/collections.js

```diff
@@ -34,13 +49,17 @@ function updateCollectionView(collectionJSON){
     }
     
     let collections = collectionJSON["kabanero collections"];
+    let cliVersion = collectionJSON["image"].split(":")[1];
+  
     collections.forEach(coll => {
         $("#collection-table-body").append(createCollRow(coll));
     });
 
     // hide loader table and show this one
     $(".table-loader").hide();
     $("#collection-table").show();
+    $(".cli-version").append(cliVersion);
+    $("#table-header-cli-version").show();
```

Comment:
We should create a new function to place the version into the page, this one should stay reserved for only the row data